src/container/srv_target.c

```diff
@@ -1750,9 +1765,13 @@ ds_cont_tgt_open(uuid_t pool_uuid, uuid_t cont_hdl_uuid,
 	D_DEBUG(DB_TRACE, "open pool/cont/hdl "DF_UUID"/"DF_UUID"/"DF_UUID"\n",
 		DP_UUID(pool_uuid), DP_UUID(cont_uuid), DP_UUID(cont_hdl_uuid));
 
+retry:
 	rc = ds_pool_thread_collective(pool_uuid, PO_COMP_ST_NEW | PO_COMP_ST_DOWN |
 				       PO_COMP_ST_DOWNOUT, cont_open_one, &arg, 0);
-	if (rc != 0)
+	if (rc != 0) {
+		if (rc == -DER_AGAIN)
+			goto retry;
```

Comment:
Busy retry? It may be better to wait for former inflight opening to return and then check again.

Reply:
this is a collective operation. which will yield, but I assume wait a few ms might be ok.